CONTRIBUTING.md

```diff
@@ -3,7 +3,38 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
-## Formatting
+## Formatting and Preparation
```

Comment:
I think we should start with `## Building` or `## Setup` section above the `## Formatting` section. The first section will explain how to install the tools necessary to view the course (what you find in the README today).